src/Tvheadend.h

```diff
@@ -26,6 +26,7 @@
 #include "platform/threads/threads.h"
 #include "platform/threads/mutex.h"
 #include "platform/util/buffer.h"
+#include "platform/util/StringUtils.h"
```

Comment:
StringUtils not used in the header file. Better move include(s) into the cpp files actually using it.